teuthology/orchestra/run.py

```diff
@@ -90,8 +90,10 @@ def execute(self):
         prefix = "Running:"
         if self.label:
             prefix = "Running ({label}):".format(label=self.label)
-        log.getChild(self.hostname).info(u"{prefix} {cmd!r}".format(
-            cmd=self.command, prefix=prefix))
+        log.getChild(self.hostname).info(prefix)
+        for line in self.command.split('\n'):
+            log.getChild(self.hostname).info('{:>{indent}} {line}'.format(
+                '>', indent=len(prefix), line=line))
```

Comment:
@tchaikov Thanks for the commit :-) please excuse me, but looks like that long indentation is overkill, especially when one would like to use _label_, check it out, the code:
```python
from teuthology.orchestra.remote import Remote as R
r=R('127.0.0.1')
r.run(label="I am not a weird set of arguments any more",
      args="""\
                for i in one two ; do
                    echo "$(( 1 + a++ )) is $i"
                done""")
```
will produce:
```
2018-10-26 21:21:08,740.740 INFO:teuthology.orchestra.run.127:Running (I am not a weird set of arguments any more):
2018-10-26 21:21:08,740.740 INFO:teuthology.orchestra.run.127:                                                    >                 for i in one two ; do
2018-10-26 21:21:08,740.740 INFO:teuthology.orchestra.run.127:                                                    >                     echo "$(( 1 + a++ )) is $i"
2018-10-26 21:21:08,740.740 INFO:teuthology.orchestra.run.127:                                                    >                 done
2018-10-26 21:21:08,785.785 INFO:teuthology.orchestra.run.127.stdout:1 is one
2018-10-26 21:21:08,785.785 INFO:teuthology.orchestra.run.127.stdout:2 is two
```
However if you use just `> ` prefix, it looks better, for example, this change:
```diff
diff --git a/teuthology/orchestra/run.py b/teuthology/orchestra/run.py
index 2f2d89c6..49266b89 100644
--- a/teuthology/orchestra/run.py
+++ b/teuthology/orchestra/run.py
@@ -92,8 +92,7 @@ class RemoteProcess(object):
             prefix = "Running ({label}):".format(label=self.label)
         log.getChild(self.hostname).info(prefix)
         for line in self.command.split('\n'):
-            log.getChild(self.hostname).info('{:>{indent}} {line}'.format(
-                '>', indent=len(prefix), line=line))
+            log.getChild(self.hostname).info('> %s' % line)
 
         if hasattr(self, 'timeout'):
             (self._stdin_buf, self._stdout_buf, self._stderr_buf) = \
```
will give you out for the same script:
```
2018-10-26 21:25:12,926.926 INFO:teuthology.orchestra.run.127:Running (I am not a weird set of arguments any more):
2018-10-26 21:25:12,926.926 INFO:teuthology.orchestra.run.127:>                 for i in one two ; do
2018-10-26 21:25:12,926.926 INFO:teuthology.orchestra.run.127:>                     echo "$(( 1 + a++ )) is $i"
2018-10-26 21:25:12,926.926 INFO:teuthology.orchestra.run.127:>                 done
2018-10-26 21:25:12,970.970 INFO:teuthology.orchestra.run.127.stdout:1 is one
2018-10-26 21:25:12,970.970 INFO:teuthology.orchestra.run.127.stdout:2 is two
```
Thanks for patience, and patches.

Reply:
@kshtsk updated and repushed.